index.html

```diff
@@ -56,49 +53,38 @@ <h2>Introduction to EVM Object Format</h2>
 									<li>
-										<span class="icon major style5 fa-file-code"></span>
-                    <h3><a href="https://eips.ethereum.org/EIPS/eip-3670">EIP-3670</a></h3>
-                    <p><b>EOF - Code Validation</b>.</br> Defines how EOF bytecode is validated for correctness at deploy time.</p>
+                    <h3>Remove dynamic JUMPs</h3>
+                    <p>EOF bytecode uses only Static Relative JUMPs, which is very desirable for tools involving static analysis and formal verification, as well as L2 EVMs. </p>
 									</li>
-								</ul>
-                <ul class="features">
 									<li>
-										<span class="icon major style2 fa-file-code"></span>
-                    <h3><a href="https://eips.ethereum.org/EIPS/eip-4200">EIP-4200</a></h3>
-                    <p><b>EOF - Static Relative Jumps</b>.</br> Introduces new instructions: RJUMP, RJUMPI, and RJUMPV with a signed immediate encoding the jump destination.</p>
+                    <h3>Code and gas non-observability</h3>
+                    <p>Lack of code and gas observability is a <a href="https://ethereum-magicians.org/t/eof-proposal-ban-code-introspection-of-eof-accounts/12113">long-sought trait of the EVM</a> allowing for more maintainable and upgradeable protocol.</p>
 									</li>
 									<li>
-										<span class="icon major style4 fa-file-code"></span>
-                    <h3><a href="https://eips.ethereum.org/EIPS/eip-4750">EIP-4750</a></h3>
-                    <p><b>EOF - Functions</b>.</br>Introduces EOF Functions as separate code sections. Also introduces two new instructions; CALLF and RETF to call and return from functions.</p>
+                    <h3>Remove JUMPDEST-analysis</h3>
+                    <p>By obsoleting JUMPDEST-analysis done in contract runtime EOF simplifies reasoning about EVM bytecode, as well as other changes like <a href="https://eips.ethereum.org/EIPS/eip-6690">EVMMAX</a>, <a href="https://verkle.info">Verkle</a> or bytecode size increase.</p>
 									</li>
 									<li>
-										<span class="icon major style7 fa-file-code"></span>
-                    <h3><a href="https://eips.ethereum.org/EIPS/eip-5450">EIP-5450</a></h3>
-                    <p><b>EOF - Stack Validation</b>.</br> Deploy-time validation of stack usage for EOF Functions.</p>
+                    <h3>Add EVM functions</h3>
+                    <p>Organizing the EVM bytecode into functions improves analysis opportunities by encoding the number of inputs and outputs for each given function, and isolating the stack of each function.</p>
 									</li>
-                </ul>
-                <ul class="features">
 									<li>
-										<span class="icon major style8 fa-file-code"></span>
-                    <h3><a href="https://eips.ethereum.org/EIPS/eip-6206">EIP-6206</a></h3>
-                    <p><b>EOF - JUMPF intruction.</b><br>Introduces JUMPF instruction for chaining function calls.</p>
+                    <h3>Code versioning</h3>
```

Comment:
As we are here maybe add "Forward-compatibility properties":

> EOF adds new forward-compatibility properties to EVM allowing introducing new features in risk-free manner.

Reply:
Are you proposing a separate bullet or expanding this one? I can't immediately see how to weave this in, can you suggest a change?

Reply:
Separate point. But we can make it some other time.